doc/get-started.md

```diff
@@ -35,8 +36,32 @@ flutter create in_app_payments_flutter_app
 See the **Create the app** step of the [Test Drive] section in Flutter's getting
 started guide for more detailed instructions.
 
-
-## Step 2: Configure the In-App Payments SDK dependency
+## Step 2: Add the In-App Payments SDK to your iOS project
+
+To use the In-App Payments plugin on iOS devices, install **In-App Payments SDK for iOS** 
+to make it an available resource for the Flutter library. 
+
+1. Open your iOS project `Runner.xcodeproj` with **Xcode** and complete 
```

Comment:
flutter plugin has In-App Payments SDK embedded, so this step is not needed.

Reply:
Removed manual build step, left instruction to open project in Xcode.